packages/embeds/embed-core/playwright/lib/testUtils.ts

```diff
@@ -38,12 +38,44 @@ export const getEmbedIframe = async ({
   page: Page;
   pathname: string;
 }) => {
+  page.on("console", (msg) => {
+    console.log(`Browser Console: ${msg.type()}: ${msg.text()}`);
+  });
+
+  page.on("framenavigated", async (frame) => {
+    console.log(`Navigation occurred in frame: ${frame.url()}`);
+  });
+
+  const handleFrameErrors = (frame: Frame) => {
+    frame.on("pageerror", (error) => {
+      console.error(`Iframe error (${frame.url()}): ${error.message}`);
+    });
+    frame.on("requestfailed", (request) => {
+      console.error(
+        `Iframe failed request (${frame.url()}): ${request.url()}, ${request.failure()?.errorText}`
+      );
+    });
+  };
+
+  page.on("pageerror", (error) => {
+    console.error(`Page error: ${error.message}`);
+  });
+
+  page.on("requestfailed", (request) => {
+    console.error(`Failed request: ${request.url()}, ${request.failure()?.errorText}`);
+  });
```

Comment:
Now the browser console logs would be available inside CI test log to be able to easily identify what was going on in the embed. This should give us a good idea why embed test failed.

Reply:
Yeah, this makes sense